### ISSUE TITLE
cookiecutter-eos

### ISSUE THREAD

**Issue by xusy2k** [EOSIO/eosio-project-boilerplate-simple#3]
I've done a [cookiecutter's](https://cookiecutter.readthedocs.io/en/latest/) template to allow create skeletons in a easy way:

```bash
pip install cookiecutter

cookiecutter --no-input gh:xusy2k/cookiecutter-eos
```

**Improvements**

* ~~I've removed `package-lock.json` because GitHub found https://nvd.nist.gov/vuln/detail/CVE-2018-3774 at package url-parse, 1.4.1~~ Added again
* I've also begin with patch to v1.2.0 (there is no more --plugin eosio::wallet_[api_]plugin) see [hooks/post_gen_project.py#15](https://github.com/xusy2k/cookiecutter-eos/blob/master/hooks/post_gen_project.py#L15) , but it raises an error when compile, so it's only valid for releases < 1.2
```
fatal error: 'eosiolib/eosio.hpp' file not found
```
* If nodeos is not working at the first time, i try again if can starts with `--replay-blockchain --hard-replay-blockchain`
[eosio_docker/scripts/init_blockchain.sh#L17](https://github.com/xusy2k/cookiecutter-eos/blob/master/%7B%7B%20cookiecutter.dir_name%20%7D%7D/eosio_docker/scripts/init_blockchain.sh#L17)

All of this work is on https://github.com/xusy2k/cookiecutter-eos

**Comment by ChristianDunst** [comment]
Thank you very much for your feedback. Please keep us up to date with your latest developments.

**Comment by rickykung** [comment]
It is awesome!

For the docker v1.2.0 problem, the team has fixed the issue in v1.2.1 (https://github.com/EOSIO/eos/pull/5261). I have also updated the scripts here. (https://github.com/EOSIO/eosio-project-boilerplate-simple/commit/718e2c1b07276b01ef2f88d67b7fee797e3e571f)

**Comment by xusy2k** [comment]
Outdated ticket
